Fixes #7526 Added icons to Multimedia menu in Note Editor

mrudultora opened the pull request:
## Purpose / Description
Added icons to Multimedia menu in Note Editor. (working for all the themes)

## Fixes
Fixes #7526 

## Approach
Added new vector icons and then used them in Popup Multimedia Menu.
**After the change the menu now looks like this.**

![img](https://user-images.githubusercontent.com/65870389/111075218-ebf90780-850c-11eb-9b60-42b7dccebe28.png)


## How Has This Been Tested?
Tested on real device (Redmi Note 7S , API 29) and on emulator also (Pixel 3a , API 21).

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

david-allison-1 commented on AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml:
```diff
@@ -1,23 +1,41 @@
-            android:title="@string/multimedia_editor_popup_image"/>
+            android:icon="@drawable/ic_popup_menu_item_image"
+            android:title="@string/multimedia_editor_popup_image"
+            app:showAsAction="always"
```
Could you explain (possibly in a comment on the file) why `app:showAsAction="always"` is required

mrudultora replied:
I tried without it, was not working. Android documentation also does not explain this. It was suggested in **SO** thread.

david-allison-1 replied:
Sounds good to me - just add an explanation & link to the SO thread so others don't have the same questions

david-allison-1 commented on AnkiDroid/src/main/res/menu/popupmenu_multimedia_options.xml:
```diff
@@ -1,23 +1,41 @@
+            android:icon="@drawable/ic_popup_menu_item_image"
+            android:title="@string/multimedia_editor_popup_image"
+            app:showAsAction="always"
+            tools:ignore="AlwaysShowAction" />
```
It might be worth moving this to an "ignore" for the file, to remove line noise

mrudultora replied:
Okay, I will do that. Actually, it was giving warning. So, I just did this. I will remove this.

david-allison-1 replied:
You (should?) be able to move `tools:ignore="AlwaysShowAction"` into the XML declaration at the top of the file - we want to get rid of warnings if they're not relevant, but having the repeated declaration makes reading the code slightly harder.

mrudultora replied:
Ohh, I got that now. I interpreted in a wrong way.

david-allison-1 reviewed (commented):
Looks fantastic. One minor request, and good to go. Thanks!!!

mrudultora commented:
@david-allison-1 I tried removing `showAsAction=true` and cleaned and rebuilded the project. It's working without that also. So, basically I have removed all the impacted lines.

david-allison-1 reviewed (approved):
Thanks again. Pending squash & merge after CI passes

mikehardy reviewed (approved):
Nice!

mikehardy commented:
Hi there!

Just a friendly notice (one per person for PRs merged in March, regardless of PR quantity) that we try to process OpenCollective payments monthly - it's time for March 2021 submissions

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

Thanks!
